apollos-church-api/src/config.js

```diff
@@ -3,9 +3,10 @@ import fetch from 'node-fetch';
 import dotenv from "dotenv/config"; // eslint-disable-line
 import ApollosConfig from '@apollosproject/config';
 
-const configName = process.env.DATABASE_URL
-  ? 'config.postgres.yml'
-  : 'config.yml';
+const configName =
+  process.env.DATABASE_URL && process.env.DATABASE_CONTENT === 'true'
```

Comment:
Instead of basing off of a env variable name (which is a bit of technical debt), could we instead just do it off the existing of the file? Try to load `config.postgres.yml` and if that fails, fallback to `config.yml`?

Reply:
I like that a lot

Reply:
Oh another thing, we have to protect against certain imports in the `server/data/index.postgres.js` file so I guess I can check for the existence of the file in there too